handbook/engineering/campaigns/goals.md

```diff
@@ -1,73 +1,53 @@
+5. Campaign creators can get campaigns merged more effectively with sourcegraph than with anybody else
+Metric: <TODO>
+(Most of it currently bucketed into Managing Campaigns.)
 
-Completing a large scale change in any organization requires coordination with and approval by, many people. Tooling is required for this process to not only be efficient, but at a certain scale, be _possible_. This includes making developers aware changesets exist, reminding them to merge changesets, or if possible, sidestepping this social dilemma entirely by enabling automatic merges. 
 
-### Problems
+6. Campaigns is GA
+Outcome: Public Pricing
+Outcome: Personas of Campaign Customers
+Outcome: 3 business cases (simple and complex use cases, on monorepos, etc)
+Outcome: <xxx> % customer base adoption and N1 revenue metric
 
-* Company workflows often require issues/tickets to accompany changes to their code bases. Campaigns does natively support issue tracking systems.
-* It is currently too difficult to find changesets which cannot or have not been merged.
-* Often owners do not prioritize merging changesets. It takes too long to nudge the owners of each changeset. 
-* We do not currently support automerge. Doing so would greatly reduce the manual effort required to merge changesets. 
+7. Campaigns onboarding is smooth and self-serve
+A metric to be defined with CE
 
-### Milestones
+8. Campaigns has early traction for OSS use
+Metric:Campaigns creates value for 3 open source project maintainers
 
-* Support creating tickets/issues alongside or instead of code changes.
-* Users can search, filter and sort changesets by check and review status, title, repository and branch.
-* A campaign owner can nudge developers to merge changesets.
-* Campaigns can leverage the automerge capabilities provided by their code host.
+## Roadmap
 
-### Outcomes
-
-* An increase in the average number of merged changesets is observed.
-* A reduction in the average amount of time to close a campaign is observed.
-* The nudge and automerge features are used by customers to reduce time to merge.
-
-# Roadmap
-
-1. **Customer outreach to improve adoption of campaigns**
-1. Better burndown charts 
-1. Allow multiple changeset in a single repository
-1. Improve previews on updates to show the delta between existing changesets and the changesets after after the campaign spec changes are applied
-1. Allow campaign to specify target branch
-1. Create campaign spec even if subset of repos have failures
-1. Versioning/releasing of src-cli with respect to sg/sg
-1. Improved documentation of src-cli login process
-1. Add filtering/searching to campaign and changeset lists
-1. Ability to auto-merge changesets, depending on various conditions
-1. Ability to add default reviewers to changesets
+<A screenshot of ProductBoard that shows the scope of product research, engineering experiments, and development leading to those metrics and outcomes>
```

Comment:
I think we will need more than a screenshot, as this section is pulled into the [product roadmap](https://about.sourcegraph.com/handbook/product/roadmap) page.